config/secrets.yml

```diff
@@ -1,6 +1,10 @@
+<% require 'securerandom' %>
+
 development:
   secret_key_base: "boop"
 test:
   secret_key_base: "boop"
 production:
-  secret_key_base: "secret_key_base is intentionally not set for this application"
+  secret_key_base: <%= SecureRandom.hex(64) %>
```

Comment:
Was the original value causing problems here? I would actually prefer it be set to something like an English sentence that explains WTF Is going on just in case someone ends up trying to use it. Setting it to an actual value makes it look like it's being used.

Reply:
How about a comment instead?

My concern with this is that this doesn't actually "unset" it, it just uses a throwaway value. The Rails 4 solution was acceptably because we could be confident that Rails was not using the value without it raising the error.

With the current solution, if in the future something else in Rails begins to use the `secret_key_base`, we won't know about it, and having it is as a well-known, hardcoded value leaves us at risk.

A comment that we don't intend to use the `secret_key_base`, so we set it to a random value on startup, should address both concerns, right?

Reply:
Yep, comment sounds good.